scripts/node_cleanup.py

```diff
@@ -13,13 +13,21 @@ def main():
     stale = query.find_stale_locks(args.owner)
     if not stale:
         return
+    by_owner = {}
+    for node in stale:
+        if args.owner and node['locked_by'] != args.owner:
+            # This should not happen
```

Comment:
Should we log this like Error: "node shouldn't be locked by other people"

Reply:
Added